src/app/app.component.less

```diff
@@ -3,10 +3,19 @@
 button.mat-primary.mat-raised-button:not(.mat-icon-button) {
   background-color: #07676c !important;
 }
-
-
+.mat-button.mat-primary,.button-icon{
+  color:  #07676c;
+}
 // end of theming
-
+.mat-mini-fab{
```

Comment:
move this styles to the scope of theming (it is defined by comments above)

Reply:
Done